app/src/main/java/org/wikipedia/dataclient/Service.kt

```diff
@@ -329,13 +329,13 @@ interface Service {
         @Query("uccontinue") uccontinue: String?
     ): Observable<MwQueryResponse>
 
-    @GET(MW_API_PREFIX + "action=query&list=usercontribs&ucprop=ids|title|timestamp|comment|size|flags|sizediff|tags")
+    @GET(MW_API_PREFIX + "action=query&list=usercontribs&ucprop=ids|title|timestamp|comment|size|flags|sizediff|tags&meta=userinfo&uiprop=groups|blockinfo|editcount|latestcontrib")
```

Comment:
This function is used for fetching lists of User Contributions, and should not be overloaded with the extra request of getting User Info in addition. We can break this out into a separate function, if there isn't an existing `userInfo` function that we can use.